src/main/java/frc/robot/subsystems/intake/BeamBreaker.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+
+
+public class BeamBreaker extends LoggedSubsystem<BeamBreakerLoggedInputs> {
```

Comment:
What is the benefit of having a separate class for the beam breaker sensor?

Reply:
a seperate class for beambreaker provides more  options for intigration.

Reply:
Please elaborate.